include/mbedtls/x509.h

```diff
@@ -406,6 +410,12 @@ int mbedtls_x509_time_is_future(const mbedtls_x509_time *from);
  */
 int mbedtls_x509_parse_subject_alt_name(const mbedtls_x509_buf *san_buf,
                                         mbedtls_x509_subject_alternative_name *san);
+/**
+ * \brief          Unallocate all data related to subject alternative name
+ *
+ * \param san      SAN structure to free
```

Comment:
Again, sorry to nitpick but this implies that we call `free()` directly on `san`, which is not true. We call free on any additional memory that `san` owns. I fear this may trip someone up in future if it's not clear.

This is the best wording I could come up with, but feel free to use a better one if you can find it:
```suggestion
 * \param san      SAN structure - extra memory owned by this structure will be freed
```

Reply:
Ah, right, now I got it :)